lib/src/widgets/switch/ds_custom_switch.widget.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+/// Animation duration for [CustomSwitch].
+@protected
+const Duration _animationDuration = Duration(milliseconds: 250);
+
+class DSCustomSwitch extends StatelessWidget {
+  /// Creates a customizable switch for use alone or in conjunction with ListTile
+  ///
+  /// Set the active and inactive colors using the parameters [activeColor] and [inactiveColor], and use [value] to turn the switch on and off.
+  const DSCustomSwitch({
```

Comment:
I don't think the name "Custom" is necessary. It's the design system switch. Change the file name also.

Reply:
Done